raiden/transfer/state_change.py

```diff
@@ -536,7 +536,7 @@ def __init__(
         if not isinstance(partner, typing.T_Address):
             raise ValueError('partner must be of type address')
 
-        self.token_network_identifier = token_network_identifier
+        self.payment_network_identifier = token_network_identifier
```

Comment:
Pls rename `token_network_identifier` as well.

Reply:
Or better question, is there a difference between `TokenNetworkIdentifier` and `PaymentNetworkIdentifier`? I thought a payment network is a set of token networks, but in tht case the type annotation would be wrong.

Reply:
>  I thought a payment network is a set of token networks, but in tht case the type annotation would be wrong.

Yep, this is the case

Reply:
> Yep, this is the case

That the annotation is wrong?